transport/server.cc

```diff
@@ -179,6 +179,10 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
                         sm::description(
                             seastar::format("Holds an incrementing counter with the requests that ever blocked due to reaching the memory quota limit ({}B). "
                                             "The first derivative of this value shows how often we block due to memory exhaustion in the \"CQL transport\" component.", _max_request_size))),
+        sm::make_derive("requests_shed", _requests_shed,
+                        sm::description(
+                            seastar::format("Holds an incrementing counter with the requests that were shed due to overload (more than {} concurrent requests per shard). "
+                                            "The first derivative of this value shows how often we shed requests due to overload in the \"CQL transport\" component.", _config.max_concurrent_requests()))),
```

Comment:
I suggest that you write the name "max_concurrent_requests_per_shard", not the value of that variable, for two reasons: First, it is useful to know how to set this thereshold. Second, and more importantly - you allow this threshold to actually change... So (IIUC) this string is built just once, with some fixed setting of this variable, and it can be changed later and the message will become out-of-date.

Reply:
done